syntaxes/template.json

```diff
@@ -31,25 +31,41 @@
       ]
     },
 
-    "propertyBinding": {
-      "begin": "(\\[.*\\])(=)(\"|')",
+    "ngBinding": {
+      "begin": "(([\\[\\(]{1,2})(?:\\s*)(@?[-_a-zA-Z0-9.$]*)(?:\\s*)([\\]\\)]{1,2}))(=)([\"'])",
       "beginCaptures": {
         "1": {
-          "name": "entity.other.attribute-name.html"
+          "name": "entity.other.attribute-name.html entity.other.ng-binding-name.html"
         },
         "2": {
-          "name": "punctuation.separator.key-value.html"
+          "name": "punctuation.definition.ng-binding-name.begin.html"
```

Comment:
What is the advantage of this custom scope name?

Reply:
@ayazhafiz It provides a hook for theme authors to give a unique token color to the punctuation, or to Angular bindings vs regular HTML attributes, etc. The `entity.other.attribute-name` scope is kept as a fallback so themes that haven't gone out of their way to color Angular bindings won't have missing colors.

For example, this is how my theme handles the test cases in this branch:

![Angular template syntax highlighting](https://i.imgur.com/yoa2uaC.png)